app/modules/intelligence/provider/provider_service.py

```
@@ -3,32 +3,23 @@
 from enum import Enum
 from typing import List, Tuple
 
-from crewai import LLM
-from langchain_anthropic import ChatAnthropic
-from langchain_deepseek import ChatDeepSeek
-from langchain_openai.chat_models import ChatOpenAI
-from portkey_ai import PORTKEY_GATEWAY_URL, createHeaders
+from langchain_community.chat_models import ChatLiteLLM
 
 from app.modules.key_management.secret_manager import SecretManager
```

Ruff (0.8.2)

`app.modules.key_management.secret_manager.SecretManager` imported but unused

Remove unused import: `app.modules.key_management.secret_manager.SecretManager`

(F401)